requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
 django==3.1
-python-decouple==3.3
\ No newline at end of file
+python-decouple==3.3
+ipdb==0.13.3
```

Comment:
ipdb don't make sense to be here

Reply:
this is a requirements dev, python-decouple and django is wrong here.
the correct is create:
 `requeriments.txt
 requirements-dev.txt
`
and inside requirements-dev `-r requirements.txt`

Reply:
`coverage==4.5.1
flake8==3.7.8
flake8-docstrings==1.3.0
ipdb==0.12.1
`

Reply:
![image](https://user-images.githubusercontent.com/16446613/91311636-b0be0380-e789-11ea-993c-af78b77fc0f2.png)